catroid/src/org/catrobat/catroid/pocketmusic/ui/NoteView.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Catroid: An on-device visual programming system for Android devices
+ * Copyright (C) 2010-2015 The Catrobat Team
+ * (<http://developer.catrobat.org/credits>)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * An additional term exception under section 7 of the GNU Affero
+ * General Public License, version 3, is available at
+ * http://developer.catrobat.org/license_additional_term
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.catrobat.catroid.pocketmusic.ui;
+
+import android.content.Context;
+import android.view.View;
+
+import org.catrobat.catroid.R;
+
+public class NoteView extends View implements View.OnClickListener {
+
+	private boolean toggled;
+
+	public NoteView(Context context) {
+		super(context);
+		setBackgroundColor(getToggledColor());
+		setOnClickListener(this);
+	}
+
+	@Override
+	public void onClick(View v) {
+		toggled = !toggled;
+		setBackgroundColor(getToggledColor());
+	}
+
+	private int getToggledColor() {
+		return toggled ? getResources().getColor(R.color.dark_gray) : getResources().getColor(R.color.white);
```

Comment:
as already written by @84n4n4 : (19 Feb)
"set up a private static final members for black and white color that contain the values from getResources().getColor."
just be aware of good names for the variables: black/white will not stay forever since UX team will probably change the buttons' appearance in near future. toggledColor/untoggledColor or similar would be better names.